web/themes/custom/pineapple/templates/content/node--page--full.html.twig

```diff
@@ -80,12 +80,19 @@
     view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
   ]
 %}
+{%
+  set title_classes = [
+  'page-title',
+  content.field_bp_hide_page_heading.0['#markup'] == 'Hide' ?
+    'visually-hidden',
```

Comment:
Put 'visually-hidden' on line 86 and also replace ".0" with `|field_value|first` it's better practice

Reply:
done